vars/launchBuild.groovy

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/groovy
+
+def call(ctx, jobName, jobId, buildName, fromImage, toTag, dockerfile) {
+  def buildSecret = ctx.dockerCfgSecret(ctx, "builder")
+  ctx.openshift.withCluster() {
+    def output = ctx.openshift.raw("new-build",
+      "--dockerfile", "${dockerfile}",
```

Comment:
@csrwng @jupierce @gmontero this currently breaks for Dockerfiles with newilines in there -- how do we correctly make this one command line arg?

Reply:
@stevekuznetsov you may need to double-encode the backslash before the newline `\\\\n`